src/app/compare/CompareItem.tsx

```diff
@@ -37,7 +37,7 @@ export default function CompareItem({
       <div className="item-name" onClick={() => itemClick(item)}>
         {item.name} <AppIcon icon={searchIcon} />
       </div>
-      <ConnectedInventoryItem item={item} />
+      <ConnectedInventoryItem item={item} onClick={() => itemClick(item)} />
```

Comment:
Maybe we should make the item popup work here?

Reply:
i would honestly love that, just not personally ready to address sheet stacking on mobile because i have no familiarity with it or vision for it